outer-loop-cli/src/devai/commands/gitlab.py

```diff
@@ -18,8 +18,11 @@
 from langchain_google_vertexai import ChatVertexAI
 from langchain_community.agent_toolkits.gitlab.toolkit import GitLabToolkit
 from langchain_community.utilities.gitlab import GitLabAPIWrapper
+  
+USER_AGENT = 'cloud-solutions/genai-outer-loop-devai'
 
-llm = ChatVertexAI(model_name="gemini-pro",
+with telemetry.tool_context_manager(USER_AGENT):
```

Comment:
@aburhan do we need to import anything for `telemetry.tool_content_manager`?

Reply:
updated